src/components/SeriesContainer.jsx

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+import SeriesCard from 'components/SeriesCard';
+
+const SeriesContainer = ({ series }) => {
+  console.log(series);
+  return (
+    <div className="Series">
+      {series &&
+        series.length &&
+        series.map((serie, index) => <SeriesCard key={index} title={serie?.full_name ?? ''} />)}
+    </div>
+  );

```

Comment:
I think this could be more explicit rather than implicit. You are checking that the series is not defined. So you could move that into a variable to make that easier to read
```suggestion
  const hasSeries = series && series.length > 0;
  return (
    <div className="Series">
      {hasSeries &&
        series.map((serie, index) => <SeriesCard key={index} title={serie.full_name ?? ''} />)}
    </div>
  );

```